src/components/Map/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { BikeRentalStation, Scooter } from '@entur/sdk'
+import React, { useState, memo } from 'react'
+
+import ReactMapGL, { Marker } from 'react-map-gl'
+
+import PositionPin from '../../assets/icons/positionPin'
+import ScooterOperatorLogo from '../../assets/icons/scooterOperatorLogo'
+
+import { useSettingsContext } from '../../settings'
+import { StopPlaceWithDepartures } from '../../types'
+
+import BikeRentalStationTag from './BikeRentalStationTag'
+import StopPlaceTag from './StopPlaceTag'
+
+import './styles.scss'
+
+const Map = ({
+    stopPlaces,
+    bikeRentalStations,
+    scooters,
+    walkTimes,
+    interactive,
+    mapStyle,
+    latitude,
+    longitude,
+    zoom,
+}: Props): JSX.Element => {
+    const [, { setZoom }] = useSettingsContext()
```

Comment:
Skulle gjerne hatt `useSettingsContext` heilt ut frå denne komponenten. Ha heller ein `onZoomChange`-prop og la forelderkomponenten bestemme om setZoom skal settast i settings.

Dersom man i framtida vil ha ein visningstype med to kart ved sida av kvarandre, så ville man gjerne brukt denne <Map />-komponenten. Då kan det bli rart om zooming i det eine kartet fører til at "global zoom" blir satt, sånn at det andre kartet også blir zooma inn/ut. Derfor er det fint om man kan kontrollere dette utanfor komponenten, og komponenten blir dum og deilig.

Reply:
Hmm, ser fordelene. Problemet akkurat nå er at kartet er en "smart" komponent som rekalkulerer clusters ved endring på `zoom`.

Jeg tar en titt på å flytte dette ut av kartkomponenten når jeg har fått clustering til å fungere ordentlig. Så kan kartet kun ta inn data og presentere det :D

Reply:
`zoom` kjem inn som prop allerede, så om du flyttar ut `setZoom` også, så endrar vel ikkje det på ting?

Reply:
Jo, men den prop'en kommer fra settings, så om ikke det blir satt ny zoom der så kalkureres clusters alltid på samme zoom nivå :/

Reply:
Jeg ser på å få fikset det nå

Reply:
Fant en løsning, fjernet bare `useSettingsContext` i denne branchen og pushet. Man kan fortsatte zoome selv i kart-visning da en onViewPortChange-event setter zoom nivå der, og i tile så blir zoom nivå fortsatt satt i admin-panelet